src/components/FormDatePicker.vue

```diff
@@ -162,6 +159,12 @@ export default {
             ? this.validator.errors.get(this.name)
             : [];
       }
+    },
+    value(valuee) {
```

Comment:
Typo in the variable name
```suggestion
    value(value) {
```

Reply:
That is actually intentional because if it is the same name, I'm creating a shadow variable with having the same function name